pkg/indexer/storer/groupMessage.go

```diff
@@ -2,23 +2,55 @@ package storer
 
 import (
 	"context"
-	"errors"
+	"fmt"
 
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/xmtp/xmtpd/pkg/abis"
 	"github.com/xmtp/xmtpd/pkg/db/queries"
 	"go.uber.org/zap"
 )
 
 type GroupMessageStorer struct {
```

Comment:
Understanding the `LogStorer` interface a little better - what are some other examples of `LogStorer` we might want, now or in the future?

Reply:
Probably just this and `IdentityUpdate`s in the near future.